allsky_mqttsubscribe/allsky_mqttsubscribe.py

```diff
@@ -0,0 +1,201 @@
+'''
+allsky_boilerplate.py
+
+Part of allsky postprocess.py modules.
+https://github.com/thomasjacquin/allsky
+
+
+'''
+import allsky_shared as s
+import json
+import os
+import time
+import paho.mqtt.client as mqtt
+
+metaData = {
+    "name": "Subscribes to MQTT topic and gets messages",
+    "description": "Obtains JSON payload from given MQTT topic for use in overlay manager",
+    "module": "allsky_mqttsubscribe",
+    "version": "v1.0.0",    
+    "events": [
+        "periodic"
+    ],
+    "experimental": "true",
+    "arguments":{
+        "extradatafilename": "allskymqttsubscribe.json",
+        "period": 60,
+        "prefix": "",
+        "mqttserver": "",
+        "mqttport": 1883,
+        "mqtttopic": "",
+        "mqttusername": "",
+        "mqttpassword": ""
+        },
+    "argumentdetails": {
+        "mqttserver": {
+            "required": "true",
+            "description": "MQTT server to connect to",
+            "help": "Example: 192.168.1.250"
+        },
+        "mqttport": {
+            "required": "true",
+            "description": "MQTT port to connect to",
+            "help": "Example: 1883"
+        },
+        "mqtttopic": {
+            "required": "true",
+            "description": "MQTT Topic to subscribe to",
+            "help": "Example: astro/NINA"
+        },
+        "mqttusername": {
+            "required": "false",
+            "description": "MQTT Username",
+            "help": "Username for MQTT server"
+        },
+        "mqttpassword": {
+            "required": "false",
+            "description": "MQTT Password",
+            "help": "Password for MQTT server"
+        },
+        "extradatafilename" : {
+            "required": "true",
+            "description": "Extra Data Filename",
+            "help": "The name for the extra variables file"
+        },
+        "period" : {
+            "required": "false",
+            "description": "Read Every",
+            "help": "Reads data every x seconds.. Zero will disable this and run the check every time the periodic jobs run",
+            "type": {
+                "fieldtype": "spinner",
+                "min": 0,
+                "max": 600,
+                "step": 1
+            }
+        }
+    },
+    "enabled": "false",
+    "changelog": {
```

Comment:
You have changelog in the meta data twice

Reply:
Thank you for the review! I fixed this.